railties/test/railties/engine_test.rb

```diff
@@ -267,6 +267,23 @@ class AddFirstNameToUsers < ActiveRecord::Migration::Current
       assert_nothing_raised { BukkitController }
     end
 
+    test "adds its draw paths to routes" do
+      @plugin.write "config/routes/testing.rb", <<-RUBY
+        Rails.application.routes.draw do
+          get "/route", to: "a#b"
+        end
+      RUBY
+
+      boot_rails
+
+      app_paths = Rails.application.routes.draw_paths.map { |path| path.relative_path_from(Rails.root).to_s }
```

Comment:
Change this test to assert the behavior, not the implementation please.

Reply:
I changed to assert the output of `bin/rails routes` with a route defined in a engine draw path. Let me know if you wanted something different.